### PR TITLE
Set new nextChargeDate for subscription made after middle of the month

### PR DESCRIPTION
**Pull request by flickz**
This PR follows up https://github.com/opencollective/opencollective-api/pull/2637 to fix [#1215](https://github.com/opencollective/opencollective/issues/1215). 

This only affect subscription made on or after 15th of each months. 

Screenshot 2019-09-30 at 10 16 44 PM

### CODE REVIEW

File: test/cypress/integration/12-contributionFlow.donate.test.js [hunk 1]
```diff
@@ -38,7 +38,8 @@ describe('Contribution Flow: Donate', () => {
       cy.contains('[data-cy="select-option"]', 'Monthly').click();
       cy.tick(1000); // Update details is debounced, we need to tick the clock to trigger update
       cy.contains('.step-details', '$1,337.00 per month');
-      cy.contains('Next charge: Jun 1, 2042');
```
Comment (znarf) [inline test/cypress/integration/12-contributionFlow.donate.test.js]: Can you make sure there is a test when donating before the 15th?

File: lib/date-utils.js [hunk 2]
```diff
@@ -9,6 +9,9 @@ import INTERVALS from './constants/intervals';
  */
 export const getNextChargeDate = (firstChargeDate, interval) => {
   if (interval === INTERVALS.month) {
+    if (firstChargeDate.getDate() >= 15) {
```
Comment (znarf) [inline lib/date-utils.js]: I would do > 15

File: test/cypress/integration/12-contributionFlow.donate.test.js (hunk 1 above)
Comment (flickz) [reply test/cypress/integration/12-contributionFlow.donate.test.js]: The test in https://github.com/opencollective/opencollective-frontend/pull/2703/files#diff-2
checks for that

**Comment by znarf** [comment]
@Betree let's ask @alanna for the right copy, what we're doing is not easy to describe.

That could be the following:

> Note: to prevent you being charged twice in a short time frame, if you contribute after the 15th of a given month, the next charge will not be the 1st of next month but the month after.

**Comment by flickz** [comment]
We also need to update the email sent to first contributors to indicate this

Screenshot 2019-10-04 at 1 25 07 PM

**Comment by znarf** [comment]
@flickz can you take care of updating this email?

**Comment by flickz** [comment]
Can I use your above copy?

**Comment by znarf** [comment]
@flickz You can try. And also feel free to try my copy in the frontend too.

**Comment by flickz** [comment]
Screenshot 2019-10-05 at 8 12 09 AM

**Comment by znarf** [comment]
@flickz could you rebase so we can merge?

**Pull request merged**